README.md

```diff
@@ -42,11 +42,33 @@ mkdir -p /var/tmp/reva/root/{home,oc}
 
 ## Quickstart
 
-After building the binary with the above steps and also setting up the prerequisites, run this command to start all services:
+After building the binary with the above steps and setting up the prerequisites
 
-```console
-./bin/ocis server
-```
+1. create a file called `identifier-registration.yml` with this content:
+
+     ```
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=5393452)